package.json

```diff
@@ -11,14 +11,20 @@
     "denali-addon",
     "addon"
   ],
+  "main": "app/addon.js",
```

Comment:
FWIW - I would love to be able to get rid of references to `dist/`, as you might have noticed by my `// TODO` comments in a couple places. It's annoying that the runtime code needs to be aware of the build system.